src/Commands/QueueFlush.php

```diff
@@ -56,6 +56,10 @@ public function run(array $params)
         $hours = $params['hours'] ?? CLI::getOption('hours');
         $queue = $params['queue'] ?? CLI::getOption('queue');
 
+        if ($hours !== null) {
+            $hours = (int) $hours;
+        }
```

Comment:
I like the ternary you switched to below. 
```suggestion
        $hours = $hours === null ? null : (int) $hours;
```